test/widgets/compose_box_test.dart

```diff
@@ -371,11 +373,22 @@ void main() {
       });
     });
 
-    testWidgets('to TopicNarrow', (tester) async {
-      await prepare(tester,
-        narrow: TopicNarrow(channel.streamId, TopicName('topic')));
-      checkComposeBoxHintTexts(tester,
-        contentHintText: 'Message #${channel.name} > topic');
+    group('to TopicNarrow', () {
+      testWidgets('with non-empty topic', (tester) async {
+        await prepare(tester,
+          narrow: TopicNarrow(channel.streamId, TopicName('topic')),
+          mandatoryTopics: false);
```

Comment:
Does `mandatoryTopics` make any difference for a topic-narrow compose box? A topic-narrow compose box doesn't offer a topic input.

Reply:
Yes, because the hint text of the content input displays the topic name (e.g.: "Message #channel > general chat").